<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvenementInter</class>
 <widget class="QDialog" name="EvenementInter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>941</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter Evenment</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>431</width>
       <height>301</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter Evenement</string>
     </property>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>150</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Emplacement">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>110</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>150</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Evenement :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Emplacement :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Nom">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Description">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="AjouterBouton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gerer Evenment</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>381</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer Evenement</string>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>17</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>12</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID evennement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SupprimerBouton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Emplacement :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Evenement :</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Nomval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Descriptionval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Emplacementval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="Dateeventval">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>170</y>
        <width>256</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tablemodifier">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>921</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>381</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Supprimer Evenement</string>
     </property>
     <widget class="QComboBox" name="comboBox_modif">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>17</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>12</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ID evennement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ModifierBouton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Emplacement :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Date Evenement :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="Emplacement_modif">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Nom_modif">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="Description_modif">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_modif">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>270</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Trier par :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>272</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Tri">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>275</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>270</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche Emplacement :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_emp">
     <property name="geometry">
      <rect>
       <x>564</x>
       <y>270</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>270</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Recherche description :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="recherche_desc">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>270</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
